prombench/Makefile

```diff
@@ -45,3 +45,28 @@ all_nodepools_deleted:
 		-v ZONE:${ZONE} -v PROJECT_ID:${PROJECT_ID} \
 		-v CLUSTER_NAME:${CLUSTER_NAME} -v PR_NUMBER:${PR_NUMBER} \
 		-f manifests/prombench/nodepools.yaml
+
+# funcbench resource
+funcbench_nodepool_create:
+	$(PROMBENCH_CMD) gke nodepool create -a ${AUTH_FILE} \
+		-v ZONE:${ZONE} -v PROJECT_ID:${PROJECT_ID} -v CLUSTER_NAME:${CLUSTER_NAME} -v PR_NUMBER:${PR_NUMBER} \
+		-f manifests/funcbench/nodepools.yaml
+
+funcbench_nodepool_delete:
+	$(PROMBENCH_CMD) gke nodepool delete -a ${AUTH_FILE} \
+		-v ZONE:${ZONE} -v PROJECT_ID:${PROJECT_ID} -v CLUSTER_NAME:${CLUSTER_NAME} -v PR_NUMBER:${PR_NUMBER} \
+		-f manifests/funcbench/nodepools.yaml
+
+funcbench_resource_apply:
+	$(PROMBENCH_CMD) gke resource apply -a ${AUTH_FILE} \
+		-v ZONE:${ZONE} -v PROJECT_ID:${PROJECT_ID} -v CLUSTER_NAME:${CLUSTER_NAME} \
+		-v PR_NUMBER:${PR_NUMBER} -v GITHUB_TOKEN:${GITHUB_TOKEN} \
+		-v GITHUB_ORG:${GITHUB_ORG} -v GITHUB_REPO:${GITHUB_REPO} \
+		-v LAST_COMMIT_SHA:${LAST_COMMIT_SHA} -v BRANCH:${BRANCH} -v REGEX:${REGEX} \
```

Comment:
Should we change `BRANCH` to `TARGET` ?

Reply:
Not sure about it. Some places (like, in `REAMD.md`) are still calling it `branch`. I think we use branch most of the time in the comments, let's keep it like this first.